### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by Ambient567**
For this pair programming project:
We collaborated with our coding partner to complete the contact form section of the portfolio.
Followed the Figma design templates for our section
And made sure there were no linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -260,5 +260,48 @@ <h3>Skills</h3>
...
+          cols="30"
+          rows="5"
+          maxlength="500"
+          placeholder="Write you message here"
+          required
+        ></textarea>
+
+        <input class="btn" type="submit" value="Get In Touch" />
+      </form>
+    </footer>
```
[the remark this answers is not in the document]
Comment (studiosnabeel) [reply index.html:305]: @youmari Hey Brother I can see the background on my PC and my partner's PC via live server. Can you please check again?

[the remark this answers is not in the document]
Comment (studiosnabeel) [reply index.html:305]: Ok brother the problem is solved I put a dot before all image pathways on HTML and CSS and now I can see all the background images and other images on my index.html file

[the remark this answers is not in the document]
Comment (christianbiring1) [reply index.html:305]: :+1:

File: style.css [hunk 2]
```diff
@@ -60,7 +60,7 @@ nav {
 
 .hero {
   background-position: fixed;
-  background-image: url("/Pictures/headershapes.png");
+  background-image: url("./Pictures/headershapes.png");
   gap: 12px;
```
Comment (christianbiring1) [inline style.css:64]: - [ ] Kindly make sure your page doesn't overlap on the right. the width must be `100%` and not more.
your design :point_down: 
image
image

File: style.css [hunk 3]
```diff
@@ -508,3 +530,95 @@ section .languages {
...
+
+footer .btn {
+  width: 140px;
+  height: 48px;
+  padding: 0.75rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  color: #6070ff;
+  background: #fff;
```
Comment (christianbiring1) [inline style.css:607]: - [ ] KIndly make sure all the from section has the correct background color. hint `You may adjust the size of your bacground`
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @studiosnabeel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
soclose

 ## Highlights
- [ ] All the form parameters meet the requirements
- [ ] all linters checked are green :heavy_check_mark: 
- [ ] Descriptive `PR` title and summary

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (studiosnabeel) [reply style.css:65]: @christianbiring1 Thank you for pointing out this issue🤗. I have adjusted the width to 100% now.

File: style.css (hunk 3 above)
Comment (studiosnabeel) [reply style.css:595]: The background color of the Contact form has been corrected. Thank you for pointing this out 🤗

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Ambient567 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**